packages/next-acms/README.md

```diff
@@ -0,0 +1,17 @@
+# next-acms package
+
+This package is added as a dependency in the next-acms-basic-starter kit which will allow for shipping updates
+in the future.
+
+To learn more about Next.js for Acquia CMS and the project's monorepo structure,
```

Comment:
I don't think we need to mention the monorepo since that's not relevant to the endusers. I think this could be simplified to something like

## Getting Started
Visit https://github.com/acquia/next-acms#readme to get started with Acquia CMS and Next.js.